.taskcluster.yml

```diff
@@ -27,15 +27,15 @@ tasks:
       maxRunTime: 14400
       image: "ubuntu:14.04"
       env:
-        TENSORFLOW_BUILD_ARTIFACT: https://index.taskcluster.net/v1/task/project.deepspeech.tensorflow.pip.v1.0.0-warpctc.gpu/artifacts/public/home.tar.xz
+        TENSORFLOW_BUILD_ARTIFACT: https://queue.taskcluster.net/v1/task/CUfR_gCURKy7IPnZzpqTxg/runs/0/artifacts/public/home.tar.xz
       command:
         - "/bin/bash"
         - "--login"
         - "-cxe"
         - >
           apt-get -qq update && apt-get -qq -y install git pixz wget pkg-config libsox-dev &&
           adduser --system --home /home/build-user build-user &&
-          cd /home/build-user/ &&
+          cd /home/ &&
```

Comment:
To simplify between OSX and Linux, the tar that was made previously from /home/build-user/ is now made from /home